word-count/word_count.py

```diff
@@ -1,2 +1,11 @@
+import re
 def count_words(sentence):
-    pass
+    #[\s,.:;!?\']?
+    list_w = [item.group(1).lower() for item in re.finditer(r'\b(([\w]+(\'s)?|[0-9]+))',sentence)]
```

Comment:
Does not \w matches numbers as well, so that [0-9] is not needed?

Also is this code capturing "they're"? Maybe I am missing it.

Reply:
\w match any letter of the alphabet, lower or uppercase. 

You are right, I skipped that case (they're)!